src/LevelDBStore/Plugins/Storage/Snapshot.cs

```diff
@@ -35,9 +36,9 @@ public void Dispose()
             snapshot.Dispose();
         }
 
-        public IEnumerable<(byte[] Key, byte[] Value)> Find(byte table, byte[] prefix)
+        public IEnumerable<(byte[] Key, byte[] Value)> Seek(byte table, byte[] keyOrPrefix, SeekDirection direction = SeekDirection.Forward)
         {
-            return db.Find(options, Helper.CreateKey(table, prefix), (k, v) => (k[1..], v));
+            return db.Find(options, Helper.CreateKey(table, keyOrPrefix), (k, v) => (k[1..], v));
```

Comment:
`direction` is not working

Reply:
I'm trying to fix this.

Reply:
@shargon `ByteArrayComparer` is internal, which I cannot use it here.

Reply:
Could you help to fix this?

Reply:
Waiting for https://github.com/neo-project/neo-modules/pull/279